CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Due to [\#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on
 ### Added
 
 - **decidim-api**: Use organization time zone [\#6088](https://github.com/decidim/decidim/pull/6088)
+- **decidim-docs**: Add helpful info to install docs for seed errors during installation process. [\#6085](https://github.com/decidim/decidim/pull/6085)
```

Comment:
Is this correct @jesusdb ? It seems this changelog entry is from different PR ?

Reply:
When I resolved conflicts with changelog, it added that line, but in files changed, I only see one addition (my PR):
![imagen](https://user-images.githubusercontent.com/61017625/81910310-149e3100-95cc-11ea-8e36-df19a8290df1.png)